### ISSUE TITLE
Why does mongo:latest point to 4.4?

### ISSUE THREAD

**Issue by dm17** [docker-library/mongo#582]
Considering the "current stable release" is v6 and not v4, why does mongo:latest point to v4.4? https://www.mongodb.com/docs/upcoming/release-notes/

**Comment by yosifkit** [comment]
:confused: It is `6.x` in every way that I can check it:
 - https://hub.docker.com/_/mongo/tags?page=1&name=latest
 - https://hub.docker.com/layers/library/mongo/latest/images/sha256-53c4915c2c13524007a84b472f8c0b0a3c4f889afcb014306b628b2e82d0b658?context=explore
 - https://github.com/docker-library/docs/blob/dd6e1ec67408ac676124b2e390b17012f6df3b22/mongo/README.md#shared-tags
 -  ```console
    $ docker pull mongo:latest
    latest: Pulling from library/mongo
    eaead16dc43b: Pull complete 
    8a00eb9f68a0: Pull complete 
    f683956749c5: Pull complete 
    b33b2f05ea20: Pull complete 
    3a342bea915a: Pull complete 
    fa956ab1c2f0: Pull complete 
    138a8542a624: Pull complete 
    acab179a7f07: Pull complete 
    f88335710e84: Pull complete 
    Digest: sha256:71a63fc2438e45714f6c8a2505968ee0beeb94ec77a88ef12190f7cee9b95f32
    Status: Downloaded newer image for mongo:latest
    docker.io/library/mongo:latest
    $ docker run -it --rm mongo:latest@sha256:71a63fc2438e45714f6c8a2505968ee0beeb94ec77a88ef12190f7cee9b95f32 mongod --version
    db version v6.0.2
    Build Info: {
        "version": "6.0.2",
        "gitVersion": "94fb7dfc8b974f1f5343e7ea394d0d9deedba50e",
        "openSSLVersion": "OpenSSL 1.1.1f  31 Mar 2020",
        "modules": [],
        "allocator": "tcmalloc",
        "environment": {
            "distmod": "ubuntu2004",
            "distarch": "x86_64",
            "target_arch": "x86_64"
        }
    }
    ```

**Comment by dm17** [comment]
Ooo does `docker-compose up -d mongo` not actually pull the latest image despite `mongo:latest` being in its docker-compose.yml? That could explain how mongo go to these states on these different systems that run that command. I wouldn't expect it to behave that way but it seems to.

**Comment by yosifkit** [comment]
If the image already exists locally, then `docker-compose up` will not pull a new one. You can add a `--pull=always` to have it always check for new versions (the default is `--pull=missing`). You should also update the compose file to have a more specific tag than `latest`; for example, so that you can control when you get major updates.

**Comment by dm17** [comment]
Yes, mongo is the one I forgot to pin. Now it makes sense; an old image existed and so it didn't update. Please disregard. Should I open a new issue to ask why there can't be an official mongo image based on alpine or scratch? Ubuntu seems like overkill for many mongo dependent applications.

**Comment by tianon** [comment]
> Should I open a new issue to ask why there can't be an official mongo image based on alpine or scratch?

https://github.com/docker-library/mongo/pull/80#issuecomment-291966876

> Ubuntu seems like overkill for many mongo dependent applications.

We chose Ubuntu because it's the most well-supported distribution by MongoDB upstream (and the one we think they would be most likely to recommend using, if they were to publish an official recommendation).

**Comment by dm17** [comment]
Feel free to delete this issue; user error.
